pagerDuty.js

```diff
@@ -0,0 +1,140 @@
+const { pdToken, pdSender, env } = require('./config.js');
+const config = require('./config.js')[env];
+const bdk = require('@salesforce/refocus-bdk')(config);
+const request = require('superagent');
+const createTTE = require('./utils/tte.js').createTTE;
+
+const { useNewPDBridge, pdBridgeUrl } = config;
+const USING_NEW_PD_BRIDGE = Boolean(useNewPDBridge && pdBridgeUrl);
+const pagerDutyServicesUrl = 'https://api.pagerduty.com/services';
+
+
+/**
+ * Create PagerDuty Trigger Event
+ *
+ * @param {String} group - Action Object
+ * @param {String} message - Salesforce Query
+ * @param {Integer} room - Room Id
+ * @returns {Promise} - PagerDuty trigger promise
+ */
+function triggerEvent(group, message, room) {
+  const obj =
+    {
+      'incident':
+        {
+          'type': 'incident',
+          'title': message,
+          'service':
+            {
+              'id': group,
+              'type': 'service_reference'
+            },
+          'body':
+            {
+              'type': 'incident_body',
+              'details': message,
+              'roomId': room
+            }
+        }
+    };
+
+  return new Promise((resolve) => {
+    request
+      .post('https://api.pagerduty.com/incidents')
+      .send(obj)
+      .set('Authorization', `Token token=${pdToken}`)
+      .set('Accept', 'application/vnd.pagerduty+json;version=2')
+      .set('From', pdSender)
+      .then((res) => {
+        resolve(res);
+      }).catch((error) => bdk.log.error(
+        'pdTriggerEvent error', error));
+  });
+}
+
+/**
+ * Query PagerDuty for services
+ *
+ * @param {Integer} offset - Amount of services to offset
+ * @returns {Promise} - PagerDuty get service promise
+ */
+function queryServices(offset) {
+  // Feature Flag
```

Comment:
@here does it make sense to remove feature flag for PD bridge now? seems to be performing well

Reply:
Remove it if it has been working well for a while.